txcache/selection.go

```diff
@@ -23,7 +24,9 @@ func (cache *TxCache) acquireBunchesOfTransactions() []bunchOfTransactions {
 }
 
 // Selection tolerates concurrent transaction additions / removals.
-func selectTransactionsFromBunches(session SelectionSession, bunches []bunchOfTransactions, gasRequested uint64, maxNum int, selectionLoopMaximumDuration time.Duration) (bunchOfTransactions, uint64) {
+func selectTransactionsFromBunches(session SelectionSession, bunches []bunchOfTransactions,
+	gasRequested uint64, maxNum int, selectionLoopMaximumDuration time.Duration,
+	selectionLoopDurationCheckInterval uint32) (bunchOfTransactions, uint64) {
```

Comment:
A little bit ambiguous - _is it milliseconds, is it microseconds?_

I know it's milliseconds. All right as it is, though :+1: